docs/book/features/block_volume.md

```diff
@@ -375,3 +376,67 @@ Cluster
             VolumeAttributes:      type=vSphere CNS Block Volume
         Events:                <none>
     ```
+
+### Statically Provision a Block Volume in Tanzu Kubernetes Grid Cluster<a id="static_volume_provisioning_tkg"></a>
+
+You can statically create a block volume in a Tanzu Kubernetes Grid (TKG) Cluster using an unused PersistentVolumeClaim (PVC) from the Project Pacific cluster only if the latter satisfies the following conditions:
+
+- It is present in the same namespace as the TKG Cluster
+
+- It is associated with the same `StorageClass` as the default storage class in the TKG Cluster
+
+- Not attached to a Pod in the Project Pacific Cluster
+
+Steps to create a static PVC in a TKG Cluster are as follows:
+
+1. Note the name of the PVC in the Project Pacific cluster
+
+2. Create a PersistentVolume with the `storageClassName` set to the default storage class of the TKG Cluster and the `volumeHandle` pointing to the value retrieved in step 1.
+
+    ```yaml
+      apiVersion: v1
+      kind: PersistentVolume
+      metadata:
+        name: static-tkg-block-pv
+        annotations:
+          pv.kubernetes.io/provisioned-by: csi.vsphere.vmware.com
+      spec:
+        storageClassName: "gc-storage-profile"  # Replace this value with the default storage class in your TKG Cluster
+        capacity:
+          storage: 2Gi
+        accessModes:
+          - ReadWriteOnce
+        persistentVolumeReclaimPolicy: Delete
```

Comment:
this should be same as the one in the SC

Reply:
This is not required.